schemas/context/environment.example.1.json

```diff
@@ -13,6 +13,9 @@
     "xdm:viewportWidth": 1680
   },
   "xdm:operatingSystem": "MAC OS",
+  "xdm:operatingSystemMinor": "OS X",
```

Comment:
This separation of system and system minor is clearly arbitrary and not based on any known standard.

Reply:
Removed Minor